src/Components/AddItem/AddItem.js

```diff
@@ -0,0 +1,146 @@
+        { value: 'Drinks', label: 'Drinks' },
+        { value: 'General', label: 'General' },
+    ];
+
+    const packeging = [
+        { value: 'Individual', label: 'Individual' },
+        { value: 'KG', label: 'KG' },
+        { value: 'Gram', label: 'Gram' },
+        { value: 'Pack', label: 'Pack' }
+    ];
+    
+    const [selectedDepartment, setSelectedDepartment] = useState(null);
+    const [selectedPackeging, setSelectedPackaging] = useState(null);
+    const [inserted, setInserted] = useState(false);
+    const [notInserted, setNotInserted] = useState(false);
+
+    const handleChange = (e) => {
+        const {name, value} = e.target;
+        setInputs({...inputs, [name]: value});
+        console.log(inputs)
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        if(!selectedDepartment || !selectedPackeging || !inputs.product)
+            return;
+
+        const {name, value} = e.target;
+        setInputs({...inputs, [name]: value})
+
+        const inputsDetails = {
+            product: inputs.product,
+            department: selectedDepartment.label,
+            packeging: selectedPackeging.label
+        }
+
+        setSelectedDepartment(null);
+        setSelectedPackaging(null);
+        setInputs({product: '', department: '', packeging: ''});
+
+        axios.post('http://localhost:5000/additem', inputsDetails)
+        .then(response => {
+            console.log(response.status);
+            if (response.data.inserted) {
+                setInserted(true)
+                setNotInserted(false)
+            }
+            else {
+                setInserted(false)
+                setNotInserted(true)
+            }})
+        .catch(error => {console.log(error, error.response)});
+    }
+
+    const alertInserted = () => {
+        return <div className="alertInserted">
+            <BsCheck2Circle className="icon" />
+            <div className="alert-text">
+                <span className="alert-title"> Success! </span> <br />
+                <span className='alert-added'> Item added successfully... </span> <br />
+            </div>
+            <AiOutlineClose className='close-alert-button-success' onClick={() => setInserted(false)}/>
+        </div>
+    }
+
+    const alertNotInserted = () => {
+        return <div className="alertNotInserted">
+            <FaExclamationCircle className="icon" />
+            <div className="alert-text">
+                <span className="alert-title"> Oops! </span> <br />
+                Something went wrong... <br />
+            </div>
+            <AiOutlineClose className='close-alert-button-oops' onClick={() => setNotInserted(false)}/>
+        </div>
+    }
+
+    return (
+        <>
+        <div className="add-item-block"> 
+            <h2>Add Item</h2>
+            <div className="add-item-explanation"> Add a new item to the database </div>
+            <form   className='addItem-form' 
```

Comment:
Please set some sort of formatting, that's difficult to read

Reply:
noted, will read about prettier